src/include/daos_srv/ras.h

```diff
@@ -40,6 +40,7 @@
 	X(RAS_ENGINE_FORMAT_REQUIRED,	"engine_format_required")	\
 	X(RAS_ENGINE_DIED,		"engine_died")			\
 	X(RAS_ENGINE_ASSERTED,		"engine_asserted")		\
+	X(RAS_ENGINE_CLOCK_DRIFT,	"engine_clock_drift")		\
```

Comment:
is there any versioning that needs to be bumped with changes to list of events?

Reply:
not currently but there probably should be, @johannlombardi should we implement versioning?